portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java

```diff
@@ -11,43 +11,47 @@
 import com.google.appengine.api.datastore.Query.SortDirection;
 
 import com.google.gson.Gson;
+//import com.google.sps.data.Task;
 import java.util.ArrayList;
+import java.util.List;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.sps.data.Comment;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 
 @WebServlet("/list-post")
 public class ListCommentServlet extends HttpServlet {
- 
+
  @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      
     response.setContentType("text/html;");
     response.setContentType("application/json;");
     Query query = new Query("Messages").addSort("timestamp", SortDirection.DESCENDING);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
-
-    String numResultsString = (request.getQueryString());
-    int reqResults = Integer.parseInt(numResultsString);
     
+    ArrayList<String> Comments = new ArrayList<String>();
```

Comment:
in java, we usually don't capitalize the first letter for variables, so "Comments" should be "comments"

Reply:
Alright, I will modify my code.